util/plancodec/id.go

```diff
@@ -0,0 +1,313 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plancodec
+
+import "strconv"
+
+const (
+	// TypeSel is the type of Selection.
+	TypeSel = "Selection"
+	// TypeSet is the type of Set.
+	TypeSet = "Set"
+	// TypeProj is the type of Projection.
+	TypeProj = "Projection"
+	// TypeAgg is the type of Aggregation.
+	TypeAgg = "Aggregation"
+	// TypeStreamAgg is the type of StreamAgg.
+	TypeStreamAgg = "StreamAgg"
+	// TypeHashAgg is the type of HashAgg.
+	TypeHashAgg = "HashAgg"
+	// TypeShow is the type of show.
+	TypeShow = "Show"
+	// TypeJoin is the type of Join.
+	TypeJoin = "Join"
+	// TypeUnion is the type of Union.
+	TypeUnion = "Union"
+	// TypeTableScan is the type of TableScan.
+	TypeTableScan = "TableScan"
+	// TypeMemTableScan is the type of TableScan.
+	TypeMemTableScan = "MemTableScan"
+	// TypeUnionScan is the type of UnionScan.
+	TypeUnionScan = "UnionScan"
+	// TypeIdxScan is the type of IndexScan.
+	TypeIdxScan = "IndexScan"
+	// TypeSort is the type of Sort.
+	TypeSort = "Sort"
+	// TypeTopN is the type of TopN.
+	TypeTopN = "TopN"
+	// TypeLimit is the type of Limit.
+	TypeLimit = "Limit"
+	// TypeHashLeftJoin is the type of left hash join.
+	TypeHashLeftJoin = "HashLeftJoin"
+	// TypeHashRightJoin is the type of right hash join.
+	TypeHashRightJoin = "HashRightJoin"
+	// TypeMergeJoin is the type of merge join.
+	TypeMergeJoin = "MergeJoin"
+	// TypeIndexJoin is the type of index look up join.
+	TypeIndexJoin = "IndexJoin"
+	// TypeIndexMergeJoin is the type of index look up merge join.
+	TypeIndexMergeJoin = "IndexMergeJoin"
+	// TypeIndexHashJoin is the type of index nested loop hash join.
+	TypeIndexHashJoin = "IndexHashJoin"
+	// TypeApply is the type of Apply.
+	TypeApply = "Apply"
+	// TypeMaxOneRow is the type of MaxOneRow.
+	TypeMaxOneRow = "MaxOneRow"
+	// TypeExists is the type of Exists.
+	TypeExists = "Exists"
+	// TypeDual is the type of TableDual.
+	TypeDual = "TableDual"
+	// TypeLock is the type of SelectLock.
+	TypeLock = "SelectLock"
+	// TypeInsert is the type of Insert
+	TypeInsert = "Insert"
+	// TypeUpdate is the type of Update.
+	TypeUpdate = "Update"
+	// TypeDelete is the type of Delete.
+	TypeDelete = "Delete"
+	// TypeIndexLookUp is the type of IndexLookUp.
+	TypeIndexLookUp = "IndexLookUp"
+	// TypeTableReader is the type of TableReader.
+	TypeTableReader = "TableReader"
+	// TypeIndexReader is the type of IndexReader.
+	TypeIndexReader = "IndexReader"
+	// TypeWindow is the type of Window.
+	TypeWindow = "Window"
+	// TypeTableGather is the type of TableGather.
+	TypeTableGather = "TableGather"
+	// TypeIndexMerge is the type of IndexMergeReader
+	TypeIndexMerge = "IndexMerge"
+	// TypePointGet is the type of PointGetPlan.
+	TypePointGet = "Point_Get"
```

Comment:
We can skip logging for PointGet actually?

Reply:
Just in case. 
And for unified style.